### PR TITLE
NGSTACK-821 sitemap separated by siteaccesses

### PR DESCRIPTION
**Pull request by ljacmi**

### CODE REVIEW

File: config/app/packages/novactive_ez_seo.yaml [hunk 1]
```diff
@@ -20,8 +20,43 @@ nova_ezseo:
...
+                description:
+                    label: 'Description'
+                    default_pattern: "<description|short_description|title|name>"
+                    maxLength: 150
+                keywords:
+                    label: 'Keywords'
+                    default_pattern: ~
+            robots:
+                sitemap:
+                    -   url: "https://%ngsite.fh_group.site_domain%/sitemap.xml"
```
Comment (emodric) [inline config/app/packages/novactive_ez_seo.yaml:56]: Extra whitespace here.

**Comment by emodric** [comment]
Not a big fan of this change due to large volume of duplicated config, but if others are for it, :+1:

**Comment by iherak** [comment]
> Not a big fan of this change due to large volume of duplicated config, but if others are for it, 👍

The reasoning is so that it is readily available in the project and harder to miss that it needs to be done for all sites :)

**Comment by emodric** [comment]
That much is clear :)

File: config/app/packages/novactive_ez_seo.yaml (hunk 1 above)
Comment (ljacmi) [reply config/app/packages/novactive_ez_seo.yaml]: Done

**Comment by emodric** [comment]
Thanks @ljacmi !

**Pull request merged**